api/v1/views/states.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python3
+"""all default RestFul API actions"""
+from api.v1.views import app_views
+from models.state import State
+from models import storage
+from flask import jsonify, abort, request
+
+
```

Comment:
Hi Gio, i think we need add this line in our code. 
Update slash options: `app.url_map.strict_slashes = False`. Into `state.py` and `index.py`